index.html

```diff
@@ -1188,10 +1190,10 @@ <h3>Multikey</h3>
 
             <p>
 In the example above, the `publicKeyMultibase` value starts with the letter `z`,
-which is the [[?MULTIBASE]] header that conveys that the binary data is
-base58-encoded using the Bitcoin base-encoding alphabet. The decoded binary data
-[[?MULTICODEC]] header is `0xed`, which specifies that the remaining data
-is a 32-byte raw Ed25519 <a>public key</a>.
+which is the <a href="#multibase-0">Multibase</a> header that conveys that the
+binary data is base-58-btc-encoded using the Bitcoin base-encoding alphabet. The
+decoded binary data header is `0xed01`, which specifies that the
+remaining data is a 32-byte raw Ed25519 <a>public key</a>.
```

Comment:
How does 0xed01 specify the format of the data?  This needs to be defined so implementers can do the same thing.